specifyweb/frontend/js_src/lib/localization/forms.ts

```diff
@@ -986,6 +986,12 @@ export const formsText = createDictionary({
     'uk-ua': 'Використовуйте локалізовані назви полів',
     'de-ch': 'Verwenden Sie lokalisierte Feldbezeichnungen',
   },
+  localizedLabels: {
+    'en-us': 'Localized Labels',
```

Comment:
since this is used as a button label, can you reword it to be something like `Display localized field labels` and `Display database field names` 

(could change `Display` to `Use` if you want)

Reply:
also, if `useFieldLabels` is no longer used, then remove that string